nuxt/server/api/auth-proxy.ts

```diff
@@ -0,0 +1,47 @@
+import { H3Event, readBody } from 'h3'
+import consola from 'consola'
+
+import { TURNSTILE_HEADER_KEY } from '~/utils/constants'
+
+export default defineEventHandler(async function (event: H3Event) {
+  const { req, res } = event.node
+  const body = await readBody(event)
+
+  switch (body.operationName) {
+    case 'authenticate':
+    case 'accountRegistration':
+      {
+        const turnstileKey = req.headers[TURNSTILE_HEADER_KEY.toLowerCase()]
+
+        if (Array.isArray(turnstileKey)) {
+          throwError(422, 'TurnstileKey cannot be an array.')
+        }
+
+        if (!turnstileKey) {
+          throwError(422, 'TurnstileKey not provided.')
+        }
+
+        const result = await verifyTurnstileToken(turnstileKey)
```

Comment:
The pipeline fails here because the `throwError`s above do not `return` so the typechecks are void at this line.

Reply:
I added the missing return statement. Let's see if the pipeline will finish now